src/main/webapp/css/srp-style.css

```diff
@@ -0,0 +1,41 @@
+.srp-container {
+  display: flex;
+}
+
+.srp {
+  flex-grow: 1;
+}
+
+.card-container {
+  margin: 20px 30px 0px 30px;
+  flex: 0 0 400px;
```

Comment:
I don't think you explicitly need to set flex-basis to 400px since the default value is auto, which will infer the 400px from its contents (e.g. the cards) which are 400px. It's usually best not to duplicate values in case they change. But of course check this first.